src/py42/util.py

```diff
@@ -72,3 +72,19 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
+def get_attribute_keys_from_class(cls):
+    """Returns attribute names for the given class object
```

Comment:
I don't think you need the word `object` here.  That kind of implies an instance is required.